lib/iris/fileformats/rules.py

```diff
@@ -24,6 +24,13 @@
 import six
 
 import collections
+from contextlib import contextmanager
```

Comment:
?  I'm confused by these ones...

Reply:
Me too.  I think that must be conflict confusion.  I'll just take that bit out.